### DISCUSSION TITLE
My remote keeps getting stuck on the lock screen

### DISCUSSION THREAD

**Discussion by liyk-master** [rustdesk/rustdesk#4637] [Feature Request]
![image](https://github.com/rustdesk/rustdesk/assets/82624310/29684323-025e-4b6c-b250-1beda3c76915)
Stuck on the lock screen interface and can't get in. After trying to open it with other remote software, it can operate normally

**Comment by liyk-master** [comment]
Wondering if anyone else has this problem?

**Reply by rivalcomp** [reply]
Same here on a random remote client machine (Windows 10, 1.2.0->1.2.0). Stuck at lock screen. If I drag up the screen, nothing happens (no login field appears), just the screen freezes for a few minutes then lockscreen appears again.

**Reply by liyk-master** [reply]
Is this a bug? Do you want to submit a bug?

**Reply by rivalcomp** [reply]
I don't know, I've just experienced it for the first time. I can only say that it surely was an elevated rustdesk process on the remote machine. I couldn't click on anything in the Task Scheduler window, and on the lock screen.

**Reply by rustdesk** [reply]
click here, and send 'ctrl+alt+del` ? If you can not see this, please try out  (controlled side) https://github.com/rustdesk/rustdesk/releases/nightly

https://github.com/rustdesk/rustdesk/issues/1244

![](https://user-images.githubusercontent.com/7346620/196513496-2271d8c9-3eef-45c7-a4ef-cacdd79dae14.png)

**Comment by 21pages** [comment]
Please install.

**Reply by liyk-master** [reply]
I installed it
